src/hydra-queue-runner/hydra-queue-runner.cc

```diff
@@ -754,6 +759,15 @@ void State::run(BuildID buildOne)
     if (!lock)
         throw Error("hydra-queue-runner is already running");
 
+    /* Set up simple exporter, to show that we're still alive. */
+    std::string metricsAddress{"127.0.0.1:" + std::to_string(metricsPort)};
+    prometheus::Exposer exposer{metricsAddress};
+    exposer.RegisterCollectable(registry);
+
+    std::cout << "Starting the Prometheus exporter, listening on "
+        << "http://" << metricsAddress << "/metrics"
```

Comment:
This is wrong in the case that we use port 0, since a random port will be picked from those available (it'll show `http://127.0.0.1:0/metrics`, I think).